src/controllers/user.js

```diff
@@ -57,65 +56,6 @@ class UserController {
     }
   };
 
-  static sendConfirmationEmail = async (email) => {
-    const user = await findUser({ email });
-
-    const secret = process.env.TOKEN_SECRET;
-    const duration = parseInt(process.env.EMAIL_TOKEN_EXPIRE, 10);
-
-    // Cache the verification token under a generated uuid key
-    const tokenId = uuidv4();
-    const data = { user: { id: user.id }, tokenId };
-    const token = generateToken(data, secret, duration);
-
-    await redis.set(
-      `${process.env.NODE_ENV}:${tokenId}`,
-      token,
-      'EX',
-      process.env.EMAIL_TOKEN_EXPIRE,
-      (err, result) => {
-        if (err) {
-          throw new Error(error.message);
-        }
-      }
-    );
-
-    // initiate email sender auth
-    const authClient = {
-      clientId: process.env.GOOGLE_MAIL_CLIENT_ID,
-      clientSecret: process.env.GOOGLE_MAIL_CLIENT_SECRET,
-      redirectUri: process.env.GOOGLE_MAIL_REDIRECT_URI,
-      refreshToken: process.env.GOOGLE_MAIL_REFRESH_TOKEN,
-    };
-
-    const result = await EmailHelper.sendEmail(
-      authClient,
-      process.env.EMAIL,
-      email,
-      confirmTemplate(`${process.env.HOST}/api/users/verify/${token}`)
-    );
-
-    if (result.envelope) {
-      return { message: 'Confirm email', token, envelope: result.envelope };
-    }
-    return result;
-  };
-
-  static resendConfirmationEmail = async (req, res) => {
-    try {
-      await findUser({ email: req.params.email });
-
-      const email = await this.sendConfirmationEmail(req.params.email);
-
-      if (!email.envelope) {
-        return res.status(502).json({ error: 'Bad Gateway' });
-      }
-      res.json(email);
-    } catch (error) {
-      res.status(404).json({ error: error.message });
-    }
-  };
-
```

Comment:
Moved sending email handlers in EmailController.js to reduce lines of code in this file and separate the logic from handlers that interact with the user directly